analysis/rogueoutlaw/src/modules/spells/RollTheBonesCounter.tsx

```diff
@@ -12,11 +12,12 @@ class RollTheBonesCounter extends Analyzer {
   static dependencies = {
     rollTheBonesCastTracker: RollTheBonesCastTracker,
   };
+  protected rollTheBonesCastTracker!: RollTheBonesCastTracker;
 
   rolltheBonesBuffDistributionChart() {
     const castTracker = this.rollTheBonesCastTracker;
 
-    const distributionObj = castTracker.rolltheBonesCastEvents.reduce((buffCount, cast) => {
+    const distributionObj = castTracker.rolltheBonesCastEvents.reduce((buffCount: any, cast) => {
```

Comment:
Should be `buffCount: RTBCast[]` right? Maybe could use a better var name too.

Reply:
Nope, `buffCount` is an indexable object of how many times a "buff level" from Roll the Bones procced. Essentially a `{[index : number] : number}`. I'll put this as the type rather than `any` to be more specific.